### PR TITLE
Add composer.json, phpcs scripts and laravel phpcs config

### PR DESCRIPTION
**Pull request by essmahr**

### CODE REVIEW

**Review by matthewlehner — CHANGES_REQUESTED** [review]
Can you add a `README` that talks about how to use this and points folks to the config file?

**Comment by essmahr** [comment]
@matthewlehner now with readme!

I made this repo public... I figured that was the path of least resistance for being able to install it inside Travis, not having to deal with auth at all

File: config/laravel_phpcs_ruleset.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
```
Comment (matthewlehner) [inline config/laravel_phpcs_ruleset.xml]: Can you rename this file to `ruleset.xml`?

File: bin/php-validate [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+phpcs --standard="$PWD/vendor/pixelunion/php-validate/config/laravel_phpcs_ruleset.xml" $1
```
Comment (matthewlehner) [inline bin/php-validate:3]: This way of getting to the config file feels weird – I think this is probably "the way" to do this with a shell script, but it seems _funny_.

Can this be done with a post install hook in the composer package that does something with this?
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Configuration-Options#setting-the-default-coding-standard

**Review by matthewlehner — APPROVED** [review]
I renamed this repo… 😬
Hope that's okay with you?

A couple of comments, but they're somewhat nitpicky if you want to ignore them.

File: bin/php-validate (hunk 2 above)
Comment (essmahr) [reply bin/php-validate:3]: going to look into this as a future PR 👍

**Pull request merged**